modules/bzip2/1.0.8.bcr.2/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "integrity": "sha256-q1oDF27hBtPw+pDjgdpHjdrkBZGBU8yiSOaCzQxKImk=",
+    "patch_strip": 0,
+    "patches": {
+        "add_build_file.patch": "sha256-QPcAoslzyuAkL4VQgb18uhZaUu8A30/LUrD8wHaKZHY=",
+        "module_dot_bazel.patch": "sha256-gwdtLOrUzwyRC3tAJJNPeB5pjz8uHlAsGXZjL/kCZOc="
+    },
+    "strip_prefix": "bzip2-1.0.8",
+    "url": "https://mirror.bazel.build/sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz"
```

Comment:
Can we use `urls` and also add the original one as backup?

Reply:
Done

Reply:
It looks like that makes CI angry, so reverted that: https://buildkite.com/bazel/bcr-presubmit/builds/10500#0194d2d8-dd70-4dfa-a738-ea77036ce41d